### ISSUE TITLE
Any plans to Banana PI?

### ISSUE THREAD

**Issue by gnomex** [rakeshpai/pi-gpio#36]
Hey guys, do you have any plans to support the [Banana Pi](http://en.wikipedia.org/wiki/Banana_Pi) GPIO ?

**Comment by tjanson** [comment]
I’m afraid pi-gpio hasn’t seen active development (by owner @rakeshpai) for almost a year. (Though small PRs have been merged.)
I myself haven’t worked with my Pi in a while either, and I don’t have a Banana Pi.

I’m guessing it wouldn’t be too difficult though and Pull Requests are of course welcome.

**Comment by rakeshpai** [comment]
Yeah, I don't get the time to work on this on a regular basis, and don't have the equipment to test. If you could send PRs though, I'll gladly merge them.
